### PR TITLE
FAQ: Describe which EU digital COVID certificates are supported - new

### PR DESCRIPTION
**Pull request by MikeMcC399**
This PR provides an overview for the EU certificates section of the FAQ describing which digital COVID certificates are supported. It is a new FAQ article.

The information was already available, but not collected in one place under the EU certificates section of the FAQ.

The PR links to each blog article for Version 2.3, 2.4 and 2.5 announcing each stage of the support for EU certificates.

The PR also links to the EU page https://ec.europa.eu/info/live-work-travel-eu/coronavirus-response/safe-covid-19-vaccines-europeans/eu-digital-covid-certificate_en providing background details, including for instance a list of all the countries which participate.

---
Which EU certs en

---
Which EU certs de

### CODE REVIEW

File: src/data/faq_de.json [hunk 1]
```diff
@@ -127,6 +127,17 @@
...
                 "accordion": [
+                    {
+                    "title": "Welche EU-Zertifikate werden unterstützt?",
+                    "anchor": "eu_dcc_which",
+                    "active": true,
+                    "textblock": [
+                        "Ab <a href='../../de/blog/2021-07-12-cwa-version-2-5' target='_blank' rel='noopener'>Version 2.5</a> der Corona-Warn-App (CWA) werden alle drei Arten der EU digitalen COVID-Zertifikate unterstützt:",
+                        "<ul>  <li>digitales Impfzertifikat</li>  <li>digitales Genesenenzertifikat</li>  <li>digitales Testzertifikat</li> </ul>",
+                        "Bisher wurden digitale Impfzertifikate erstmalig mit <a href='../../de/blog/2021-06-10-cwa-version-2-3' target='_blank' rel='noopener'>Version 2.3</a> unterstützt, gefolgt von der zusätzlichen Unterstützung von digitalen Testzertifikate in <a href='../../de/blog/2021-06-24-cwa-version-2-4' target='_blank' rel='noopener'>Version 2.4.</a> Mit der Freigabe der <a href='../../de/blog/2021-07-12-cwa-version-2-5' target='_blank' rel='noopener'>Version 2.5</a> wurden digitale Genesenenzertifikate zusätzlich unterstützt.",
```
Comment (Ein-Tim) [inline src/data/faq_de.json:137]: "Freigabe der Version 2.5" -> "Veröffentlichung von Version 2.5"

File: src/data/faq_de.json [hunk 2]
```diff
@@ -127,6 +127,17 @@
...
+                        "Die Verordnung über das <a href='https://ec.europa.eu/info/live-work-travel-eu/coronavirus-response/safe-covid-19-vaccines-europeans/eu-digital-covid-certificate_de' target='_blank' rel='noopener noreferrer'>COVID-Zertifikat der EU</a> ist am 1. Juli 2021 in Kraft getreten. Das digitale COVID-Zertifikat der EU wird in allen EU-Mitgliedstaaten anerkannt."
```
Comment (Ein-Tim) [inline src/data/faq_de.json:138]: Add "digitale"

```suggestion
                        "Die Verordnung über das <a href='https://ec.europa.eu/info/live-work-travel-eu/coronavirus-response/safe-covid-19-vaccines-europeans/eu-digital-covid-certificate_de' target='_blank' rel='noopener noreferrer'>digitale COVID-Zertifikat der EU</a> ist am 1. Juli 2021 in Kraft getreten. Das digitale COVID-Zertifikat der EU wird in allen EU-Mitgliedstaaten anerkannt."
```

**Review by Ein-Tim — CHANGES_REQUESTED** [review]
LGTM, only two things.

File: src/data/faq_de.json (hunk 1 above)
Comment (MikeMcC399) [reply src/data/faq_de.json]: You're right that "Veröffentlichung" has been used in the past, so I will align to this term.

File: src/data/faq_de.json (hunk 2 above)
Comment (MikeMcC399) [reply src/data/faq_de.json]: Thanks for spotting the missing word!

**Review by Ein-Tim — APPROVED** [review]
Thanks for taking my feedback into account! LGTM now!

**Comment by dsarkar** [comment]
@MikeMcC399 Great ! Thanks.

File: src/data/faq_de.json [hunk 3]
```diff
@@ -127,6 +127,17 @@
...
                 "accordion": [
+                    {
+                    "title": "Welche EU-Zertifikate werden unterstützt?",
+                    "anchor": "eu_dcc_which",
+                    "active": true,
+                    "textblock": [
+                        "Ab <a href='../../de/blog/2021-07-12-cwa-version-2-5' target='_blank' rel='noopener'>Version 2.5</a> der Corona-Warn-App (CWA) werden alle drei Arten der EU digitalen COVID-Zertifikate unterstützt:",
+                        "<ul>  <li>digitales Impfzertifikat</li>  <li>digitales Genesenenzertifikat</li>  <li>digitales Testzertifikat</li> </ul>",
+                        "Bisher wurden digitale Impfzertifikate erstmalig mit <a href='../../de/blog/2021-06-10-cwa-version-2-3' target='_blank' rel='noopener'>Version 2.3</a> unterstützt, gefolgt von der zusätzlichen Unterstützung von digitalen Testzertifikate in <a href='../../de/blog/2021-06-24-cwa-version-2-4' target='_blank' rel='noopener'>Version 2.4.</a> Mit der Veröffentlichung der <a href='../../de/blog/2021-07-12-cwa-version-2-5' target='_blank' rel='noopener'>Version 2.5</a> wurden digitale Genesenenzertifikate zusätzlich unterstützt.",
```
Comment (DerVogel2020) [inline src/data/faq_de.json:137]: Typo: ... von digitalen Testzertifikate**n** in ...

**Comment by DerVogel2020** [comment]
Too late 😁

File: src/data/faq_de.json (hunk 3 above)
Comment (MikeMcC399) [reply src/data/faq_de.json:137]: @DerVogel2020 
Thanks for spotting the typo. I will submit a new PR to correct it.

**Comment by MikeMcC399** [comment]
@DerVogel2020 
> Too late 😁

Unfortunately this PR was merged quite quickly, so your comment did come too late for this PR, however I submitted #1527 to correct the typo you noticed, so thanks very much for your valuable contribution! 🙂

**Comment by dsarkar** [comment]
@DerVogel2020  @MikeMcC399 Thanks to both of you.

**Pull request merged**